templates/sponsorship_status_emails/sponsorship_application_received.txt

```diff
@@ -0,0 +1,13 @@
+Subject: Sponsorship Application Received for PyLadiesCon
+
+Dear [Applicant Name],
+
+Thank you for your application for sponsorship! We have received your application and are currently reviewing it.
+
+Please allow us a few days to evaluate your application. We will notify you once a decision has been made.
```

Comment:
Perhaps we can include their application info as part of this email, this will allow them to review and have a copy of their own sponsorship application.

We can also include a short message saying "if any time you would like to update your infomation go to {{edit_url}}".